db/seeds.rb

```diff
@@ -3,7 +3,7 @@
     photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
     bio: 'Fullstack Web developer from Uganda', posts_counter: 0)
     
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221368251-1a4b9b94-80f5-48de-8f15-fafc6e397ae1.png)

- [ ] Kindly make these sub section look something like below to make your readme file look professional and descriptive :+1: 

## Install
```
bundle install
rails db:create
rails db:migrate
```

## Usage

```
rails server
```

## Run tests

```
rspec spec
```